packages/next/bin/next.ts

```diff
@@ -11,6 +11,12 @@ import arg from 'next/dist/compiled/arg/index.js'
   }
 })
 
+const React = require('react')
+
+if (typeof React.Suspense === 'undefined') {
+  throw new Error(`The version of React you are using doesn't appear to support 'Suspense' which we rely on for Next.js. Please upgrade to a version of that matches the React peerDependencies.`)
```

Comment:
```suggestion
  throw new Error(`The version of React you are using is lower than the minimum required version needed for Next.js. Please upgrade "react" and "react-dom": "npm install --save react react-dom"`)
```

Reply:
I think this should also have a `err.sh` link.

Reply:
Probably good to note that this works for `next` and `next build` / `next export` but not for when a custom server is used, since building fails with this error we could add this same message to `next-dev-server.js` only, no need to make the production server bigger.